File: 1768.cpp
#include <iostream>
#include <algorithm>

using namespace std;

class Solution {
public:
    string mergeAlternately(string word1, string word2) {
        int bound = min(word1.size(), word2.size());
        string merged = "";
        for(int i = 0; i < bound; ++i) {
            merged += word1[i];
            merged += word2[i];
        }
        while(bound < word1.size()) {
            merged += word1[bound];
            bound++;
        }
        while(bound < word2.size()) {
            merged += word2[bound];
            bound++;
        }
        return merged;
    }
};

File: 1768.py
class Solution:
    def mergeAlternately(self, word1: str, word2: str) -> str:
        bound = min(len(word1), len(word2))
        merged = ""
        for i in range(bound):
            merged += word1[i] + word2[i]
        if bound == len(word1):
            return merged + word2[bound:]
        else:
            return merged + word1[bound:]
